packages/db/tests/translate/test-collections.js

```diff
@@ -1,6 +1,38 @@
 const test = require('ava');
 const { removeNilProperties } = require('@cumulus/common/util');
-const { translateApiCollectionToPostgresCollection } = require('../../dist/translate/collections');
+const {
+  translateApiCollectionToPostgresCollection,
+  translatePostgresCollectionToApiCollection,
+} = require('../../dist/translate/collections');
+
+const {
+  fakeCollectionRecordFactory,
+} = require('../../dist');
+test('translatePostgresCollectionToApiCollection converts Postgres collection to API', (t) => {
+  const meta = '{"tag_key":"tag_value"}';
+  const tags = '["tag1","tag2"]';
```

Comment:
Since in reality these values come back from PG as an object/array and not strings, we should mock them accordingly

Reply:
Fair. 4a975bafc